src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -103,6 +103,13 @@
   <Node Name="BoundAwaitableValuePlaceholder" Base="BoundValuePlaceholderBase">
   </Node>
 
+  <!--
+  This node is used to represent an expression returning value of a certain type.
+  It is used to perform intermediate binding, and will not survive the local rewriting.
+  -->
+  <Node Name="BoundDisposableValuePlaceholder" Base="BoundValuePlaceholderBase">
```

Comment:
>BoundDisposableValuePlaceholder [](start = 14, length = 31)

Thanks.
Please add this node to `LocalRewritingValidator` too (this will assert that this node cannot survive past local rewriting)